client/src/components/previews/OrganizationPreview.js

```diff
@@ -128,6 +154,18 @@ const OrganizationPreview = ({ className, uuid }) => {
           value={Organization.humanNameOfType(organization.type)}
         />
 
+        {showLeadingPositions(
+          organization.positions,
+          PositionRole.LEADER.toString(),
+          "Leaders"
```

Comment:
Don't hard-code this, use something like `pluralize(utils.titleCase(PositionRole.LEADER.humanNameOfRole()))` instead.

Reply:
OK